### PR TITLE
Update version to 2.0.1

### LINKED ISSUE #101: Create release version 2.0.1
## Is your feature request related to a problem? Please describe.

- Update chart version in Chart.yaml to 2.0.1
- Update CHANGELOG.md accordingly
- Add minor updates to release instrux in CONTRIBUTING.md

## Describe the solution you would like

The above is done.

### PR DESCRIPTION
**Pull request by diverdane**
### What does this PR do?
- Updates chart version in Chart.yaml to 2.0.1
- Updates CHANGELOG.md accordingly
- Adds minor updates to release instrux in CONTRIBUTING.md

### What ticket does this PR close?
Resolves #101

#### Change log
- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [x] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
+
+### Changed
+
+### Fixed
```
Comment (sgnn7) [inline CHANGELOG.md:13]: These empty headers can be removed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,7 +15,9 @@ any additional information for contributing code to this project.
 
 To release a new version of this chart:
 - Make the appropriate changes
-- Update the version number
+- Update the version number in conjur-oss/Chart.yaml
```
Comment (sgnn7) [inline CONTRIBUTING.md:18]: Wrap the path in backticks and make it a link to the file for easier finding

**Review by sgnn7 — CHANGES_REQUESTED** [review]
@diverdane 2 tiny comments

File: CONTRIBUTING.md (hunk 2 above)
Comment (diverdane) [reply CONTRIBUTING.md]: Fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (diverdane) [reply CHANGELOG.md]: Fixed.

**Comment by diverdane** [comment]
@sgnn7 updated according to your comments, and added a `Deprecated` section for the `CHANGELOG.md` (with entry for `rbac.create`).

**Review by sgnn7 — APPROVED** [review]
@diverdane LGTM!

**Pull request merged**